tests/sentry/integrations/jira/test_integration.py

```diff
@@ -418,6 +422,80 @@ def get_client():
                 'updatesForm': True,
             }]
 
+    def test_get_create_issue_config_with_default_and_param(self):
+        org = self.organization
+        self.login_as(self.user)
+        group = self.create_group()
+        self.create_event(group=group)
+
+        integration = Integration.objects.create(
+            provider='jira',
+            name='Example Jira',
+        )
+        org_integration = integration.add_organization(org, self.user)
+        org_integration.config = {
+            'project_issue_defaults': {
+                six.text_type(group.project_id): {'project': '10001'}
+            }
+        }
+        org_integration.save()
+        installation = integration.get_installation(org.id)
+
+        def get_client():
+            return MockJiraApiClient()
+
+        with mock.patch.object(installation, 'get_client', get_client):
+            fields = installation.get_create_issue_config(group, params={'project': '10000'})
+            for field in fields:
+                if field['name'] == 'project':
+                    project_field = field
+                    break
```

Comment:
You could do this loop with a list comprehension

```python
project_field = [field for field in fields if field['name'] == 'project'][0]
```

Reply:
:o didn't think of that. Neat!